backend/src/server.ts

```diff
@@ -35,6 +40,25 @@ app.post('/api/saveFile', async (req, res) => {
 	}
 });
 
+
+app.get('/api/mods/assets/path', async (_, res) => {
```

Comment:
I'd prefer `'/api/mods'` because it's shorter and already makes sense (`GET` me the `mods`)